include/swift/AST/Attr.def

```diff
@@ -254,33 +261,38 @@ DECL_ATTR(_specialize, Specialize,
   33)
 SIMPLE_DECL_ATTR(objcMembers, ObjCMembers,
   OnClass |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
+  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIBreakingToRemove,
   34)
 CONTEXTUAL_SIMPLE_DECL_ATTR(__consuming, Consuming,
   OnFunc | OnAccessor |
-  DeclModifier |
-  UserInaccessible |
-  NotSerialized |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove, 40)
+  DeclModifier | UserInaccessible | NotSerialized |
+  ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd |
+  APIBreakingToRemove,
+  40)
 CONTEXTUAL_SIMPLE_DECL_ATTR(mutating, Mutating,
   OnFunc | OnAccessor |
-  DeclModifier |
-  NotSerialized |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove, 41)
+  DeclModifier | NotSerialized |
+  ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd |
+  APIBreakingToRemove,
+  41)
 CONTEXTUAL_SIMPLE_DECL_ATTR(nonmutating, NonMutating,
   OnFunc | OnAccessor |
   DeclModifier |
   NotSerialized |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove, 42)
+  ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd |
+  APIBreakingToRemove,
+  42)
 CONTEXTUAL_SIMPLE_DECL_ATTR(convenience, Convenience,
   OnConstructor |
-  DeclModifier |
-  NotSerialized |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove, 43)
+  DeclModifier | NotSerialized |
+  ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd |
```

Comment:
Only for open classes, though.

Reply:
Ah, not quite, not if you have an extension of the class.